plugins/GrapesJsBuilderBundle/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "mautic/grapes-js-builder-bundle",
+  "description": "GrapesJS Builder with MJML support for Mautic",
```

Comment:
@npracht should i remove this file?
Typically we don't keep composer.json inside bundles.
There are similar other files for phpunit.xml, phpstan.neon etc.